File: PegasusV1/PegasusV1/Entities/ModulosPerfiles.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace PegasusV1.Entities
{
    [Table("MODULOS_PERFILES")]
    public class ModulosPerfiles
    {
        public int Id { get; set; }

        [ForeignKey("Id_Modulo")]
        public Modulos? Modulo { get; set; }
        public int? Id_Modulo { get; set; }

        [ForeignKey("Id_Perfil")]
        public Perfiles? Perfil { get; set; }
        public int? Id_Perfil { get; set; }
    }
}

File: pegasusapp/src/models/modulosPerfiles.ts
import { Perfiles } from './perfiles';
import { Modulos } from './modulos';

export interface ModulosPerfiles {
  id: number;

  modulo?: Modulos | null;
  id_Modulo?: number | null;

  perfil?: Perfiles | null;
  id_Perfil?: number | null;
}
